packages/deployment/app/config.yml

```diff
@@ -143,7 +143,7 @@ default:
 
     tasks:
       AsyncOperation:
-        image: cumuluss/async-operation:26
+        image: cumuluss/async-operation:27
```

Comment:
Not that anyone is deploying with Kes, but would this version be backwards compatible with a Kes deployment?

Reply:
Yes.  But not with a kes node-8 deployment.